README.md

```diff
@@ -132,27 +132,81 @@ Load
 
 **Note about KAFKA benchmark:**
 
-Command line example:
+Command line example of KAFKA benchmark:
 ```
 ./bin/kafka-run-class.sh \
 org.apache.kafka.tools.ProducerPerformance --throughput=-1 \
 --topic=test-one \
---num-records=50000000 \
---record-size=100 \
+--num-records=1000000 \
+--record-size=1 \
 --producer-props bootstrap.servers=localhost:9092 \
-buffer.memory=67108864 \
-batch.size=8196
+buffer.memory=33554432 \
+batch.size=200
+
 ```
 Result:
 ```
-50000000 records sent, 
-68460.889663 records/sec (6.53 MB/sec), 
-8772.19 ms avg latency, 
-29552.00 ms max latency, 
-8072 ms 50th, 
-16228 ms 95th, 
-26685 ms 99th, 
-28510 ms 99.9th.
+1000000 records sent, 
+15973.420229 records/sec (0.02 MB/sec),
```

Comment:
it was expected higher rate for the smaller records
however, it is lower
is it because of the lower batch size value?
also, 1M of records seems to be to small

Reply:
@akurilov , If I set the number of records more than a million, then Kafka benchmark will throw the exception java.lang.OutOfMemoryError. Yes, it is because of the lower batch size value. Now I understand how to avoid 
 OutOfMemoryError. But if batch size = 100 and number of records = 1000000, then Kafka benchmark will throw the timeout exception. 
Before Kafka benchmark threw the exception OutOfMemoryError with batch size = 200 and number of records > 1000000. Now with such parameters Kafka works.